tests/tool-postgresql.yaml

```diff
@@ -4,10 +4,6 @@
   assert:
   - status == 0
   - stdout.indexOf("(PostgreSQL) 12") != -1
-- desc: it should find data folder
-  command: [ls, /workspace/.pgsql/data]
-  assert:
-  - status == 0
```

Comment:
It seems wrong to me, to remove this test, without adding a new one.

May I ask you to add a test, asserting that the underlying data files for the database exist?

Reply:
Hey @kylos101, I feel that this test is unnecessary since `pg_ctl` does that check anyways.
<img width="539" alt="image" src="https://user-images.githubusercontent.com/39482679/157229657-0ddff1b9-e8fa-4572-bc37-a638fed36232.png">

`pg_ctl` is invoked from the `.bashrc.d` hook
https://github.com/gitpod-io/workspace-images/blob/8f84ceddab51fa26d4f5ff53dd23b27acb929e9e/tests/tool-postgresql.yaml#L8